cccat/events/event_handlers.py

```diff
@@ -25,7 +27,8 @@ def handle_ping(data: dto.JSON) -> None:
 def handle_cat_created(data: dto.JSON) -> None:
     event_id = data.get("event_id")
     cat_id = data.get("cat_id")
-    required_keys = {"cat_id"}
+    partial_update_cat = data.get("partial_update_cat")
```

Comment:
[question] Why is there a dependency on creating an cat and updating a cat? 
What does adding a required payload for this event mean for the side that emits an event and how does this fit in the pub/sub pattern?